Loop/View Controllers/TestingScenariosTableViewController.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  TestingScenariosTableViewController.swift
+//  Loop
+//
+//  Created by Michael Pangburn on 4/20/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+import LoopKitUI
+
+
+final class TestingScenariosTableViewController: RadioSelectionTableViewController {
+
+    private let scenariosManager: TestingScenariosManager
+
+    private var scenarioURLs: [URL] = [] {
+        didSet {
+            options = scenarioURLs.map { $0.deletingPathExtension().lastPathComponent }
+            if isViewLoaded {
+                DispatchQueue.main.async {
+                    self.updateLoadButtonEnabled()
+                    self.tableView.reloadData()
+                }
+            }
+        }
+    }
+
+    override var selectedIndex: Int? {
+        didSet {
+            updateLoadButtonEnabled()
+        }
+    }
+
+    private lazy var loadButtonItem = UIBarButtonItem(title: "Load", style: .done, target: self, action: #selector(loadSelectedScenario))
```

Comment:
Why a variable? The only place it is used is to call `isEnabled`, which could just be done via the `navigationItem.rightBarButtonItem`. It is fine as-is, I'm more curious if this is a best practice. That is, if referencing a button elsewhere should it be an explicitly named variable (versus the generic and potentially unclear `navigationItem.rightBarButtonItem`)?

Reply:
It's clearer as a named button item, IMO. If you're reading the code, there's less to think about when seeing `loadButtonItem.isEnabled` versus `navigationItem.rightBarButtonItem.isEnabled` when parsing what's happening.
The other benefit is that if ever an additional right bar button item were added, explicit naming reduces the likelihood of mixing up the two.